opencti-platform/opencti-front/src/private/components/common/lists/ListFilters.jsx

```diff
@@ -63,7 +63,8 @@ const ListFilters = ({
     ? availableFilterKeys
       .map((key) => {
         const subEntityTypes = filterKeysMap.get(key)?.subEntityTypes ?? [];
-        const isFilterKeyForAllTypes = subEntityTypes.some((subType) => entityTypes.includes(subType));
+        const isFilterKeyForAllTypes = (entityTypes.length === 1 && subEntityTypes.some((subType) => entityTypes.includes(subType)))
```

Comment:
Not sure to understand this condition : isn't it enough to check that entity types are all included in subEntityTypes ? 
`entityTypes.length > 0 && entityTypes.every((subType) => subEntityTypes.includes(subType))`

But I don't really understand exactly what is `isFilterKeyForAllTypes`.

Reply:
No it's not enough, because we can have : 
entityTypes = ['Stix-Core-Object']

and subEntityTypes = ['Stix-Core-Object', 'Stix-Domain-Object', 'City', etc]

Reply:
so in this case `isFilterKeyForAllTypes` is true ? what is `isFilterKeyForAllTypes` ?

Reply:
if the filter key is for all the entity types of the screen, the filter key appears in the 'most used filters' section of the filters list

Reply:
![image](https://github.com/OpenCTI-Platform/opencti/assets/75783086/31ba4dc7-51fe-4252-96a6-1003a9782f86)

Reply:
else, it appears in the 'all other filters' section: 
![image](https://github.com/OpenCTI-Platform/opencti/assets/75783086/6cee942f-0ca4-4849-aefb-b42633344add)

Reply:
There is still something I don't understand : `Author` is not listed in most used filters in victimology, but in all other filters : 
![image](https://github.com/OpenCTI-Platform/opencti/assets/668192/55115064-321e-40d7-8da3-b07a33c26abd)

But in data => entities we can see `Author` in most used filters.
![image](https://github.com/OpenCTI-Platform/opencti/assets/668192/13882691-8ee1-43a4-bea6-a712c22a1966)
which is expected since Author is a common property for all stix objects.

Reply:
fixed :)